crates/next-dev/tests/integration/next/font-google/basic/__google_fonts__/css2?family=Inter:ital,wght@0,100..900&display=optional

```diff
@@ -0,0 +1,63 @@
+/* cyrillic-ext */
```

Comment:
Looks like this filename is invalid on Windows...

Reply:
Ugh yeah afaik : is not allowes char in Windows.

Reply:
I wouldn't put any special chars in the filename (except for `_` and `-`)